hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleTopic.java

```diff
@@ -50,6 +49,7 @@
 
 import static com.hedera.services.utils.EntityIdUtils.asAccount;
 import static com.hedera.services.utils.EntityIdUtils.asLiteralString;
+import static com.swirlds.common.CommonUtils.hex;
```

Comment:
Another `com.hedera.services.legacy.proto.utils.CommonUtils` is also being used by this class. That was why I used an `import static` here to avoid collision. Can change to using a full path of `com.swirlds.common.CommonUtils.hex` on line 106 too.